.travis.yml

```diff
@@ -69,7 +69,7 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs,victoria_sensor_msgs
     - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
```

Comment:
Huh.  So, then why did travis pass?  This file should have failed with missing includes.